README.md

```diff
@@ -10,4 +10,11 @@ Describe how you approached to problem, and what tools and techniques you used t
 
```

Comment:
When you find the time you could add some information to the Readme. For instance you could add this sentence to the first part of the Readme:
This is a backend project with API:s built using Express.